### DISCUSSION TITLE
How to solve can't find CDM files in widevine folder???

### DISCUSSION THREAD

**Discussion by ardentsoul22** [crunchy-labs/crunchy-cli#441] [Q&A]
Hello, well the title says it all, does anyone know how to solve it?
I leave screenshot
![Captura de pantalla 2024-06-21 122127](https://github.com/crunchy-labs/crunchy-cli/assets/170395372/45957f9a-8205-42ef-af1d-282fc92220c2)

**Comment by Frooastside** [comment]
this not the correct github page. you are not using crunchy-cli in that screenshot

**Comment by ardentsoul22** [comment]
sorry :v
